src/components/renderer/form-requests.vue

```diff
@@ -110,6 +111,16 @@ export default {
               record.status = this.formatStatus(record.status);
             }
             this.tableData = response.data;
+            this.countResponse = Object.keys(this.tableData.data).length;
```

Comment:
the code is duplicate in two file, create a mixin file

Reply:
Each count is related to its own component, a mixin file is not needed.